### ISSUE TITLE
Incompatible with atom-ide-ui

### ISSUE THREAD

**Issue by madc** [john-lioneil/atom-arc-dark-ui#2]
atom-ide-ui throws the following error when used with this theme:

```
Failed to activate the atom-ide-ui package

At variable @font-size is undefined in /Users/chris/.atom/packages/atom-ide-ui/modules/nuclide-commons-ui/styles/CodeSnippet.less:10:15

LessError: variable @font-size is undefined
  at /packages/atom-ide-ui/modules/nuclide-commons-ui/styles/CodeSnippet.less:10:15
```

See here: https://github.com/facebookarchive/atom-ide-ui/issues/14

**Comment by lioneil** [comment]
Fixed in [v0.1.7](https://github.com/lioneil/atom-arc-dark-ui/tree/v0.1.7) from PR #1

**Comment by madc** [comment]
Shame on me, I did not check the old issues. Sorry.
Update just came through, everything fine.
Thanks!
